config/config.go

```diff
@@ -30,11 +32,14 @@ type NodeConfig struct {
 	P2P     P2PConfig
 	RPC     RPCConfig
 	// parameters below are Rollkit specific and read from config
-	Aggregator         bool `mapstructure:"aggregator"`
 	BlockManagerConfig `mapstructure:",squash"`
 	DALayer            string `mapstructure:"da_layer"`
 	DAConfig           string `mapstructure:"da_config"`
 	Light              bool   `mapstructure:"light"`
+
+	// Sequencer-specific settings
+	Aggregator           bool `mapstructure:"aggregator"`
+	ProgressiveSequencer bool `mapstructure:"progressive_sequencer"`
```

Comment:
i think we have to settle for one name. either aggregator or sequencer. otherwise, it will be confusing. if you don't want to change Aggregator then you can use ProgressiveAggregator.

Reply:
chose Aggregator